hoomd/pytest/test_snapshot.py

```diff
@@ -125,6 +125,18 @@ def test_configuration(s):
         assert s.configuration.dimensions == 3
 
 
+def test_wrap(s):
+    if s.communicator.rank == 0:
+        s.configuration.box = [1, 1, 1, 0, 0, 0]
+        # set up particles outside of the box
+        s.particles.N = 2
+        s.particles.position[:] = [[1.5, 0, 0], [0.3, 0.2, 0]]
+        # wrap
+        s.wrap()
+        # check the particles are where they should be
+        assert numpy.all(s.particles.position == [[0.5, 0, 0], [0.3, 0.2, 0]])
```

Comment:
```suggestion
        numpy.testing.assert_allclose(s.particles.position, [[0.5, 0, 0], [0.3, 0.2, 0]])
```
`assert_allclose` will pretty-print on failure. It also handles cases where floating point numbers may be slightly different due to round off error. I'm surprised this test works as is because 0.3 and 0.2 are not exactly representable in binary floating point.

Reply:
Thanks for this suggestion!